packages/serialization/json/src/jsonParseNode.ts

```diff
@@ -11,10 +11,10 @@ export class JsonParseNode implements ParseNode {
 	constructor(private readonly _jsonNode: unknown) {}
 	public onBeforeAssignFieldValues: ((value: Parsable) => void) | undefined;
 	public onAfterAssignFieldValues: ((value: Parsable) => void) | undefined;
-	public getStringValue = () => (typeof this._jsonNode === "string" ? (this._jsonNode as string) : undefined);
+	public getStringValue = () => (typeof this._jsonNode === "string" || this._jsonNode == null  ? this._jsonNode : undefined);
 	public getChildNode = (identifier: string): ParseNode | undefined => (this._jsonNode && typeof this._jsonNode === "object" && (this._jsonNode as { [key: string]: any })[identifier] !== undefined ? new JsonParseNode((this._jsonNode as { [key: string]: any })[identifier]) : undefined);
-	public getBooleanValue = () => (typeof this._jsonNode === "boolean" ? (this._jsonNode as boolean) : undefined);
-	public getNumberValue = () => (typeof this._jsonNode === "number" ? (this._jsonNode as number) : undefined);
+	public getBooleanValue = () => (typeof this._jsonNode === "boolean" || this._jsonNode == null  ? this._jsonNode : undefined);
```

Comment:
can you add unit tests for the null case please?

Reply:
added